projects/cdk/constants/parent-animation.ts

```diff
@@ -1,8 +1,5 @@
-import {animateChild, query, style, transition, trigger} from '@angular/animations';
+import {animateChild, query, transition, trigger} from '@angular/animations';
 
 export const TUI_PARENT_ANIMATION = trigger(`tuiParentAnimation`, [
-    transition(`* => void`, [
```

Comment:
It's actually is animatable and is necessary, I believe in some cases. Do not merge this until I investigate.

Reply:
But the overflow property can't be animated with CSS :((
https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_animated_properties

Reply:
It's not on the list, but it works, at least in Chrome:
https://codepen.io/waterplea/pen/PoRJEvP

I believe we need it so scrollbars don't appear in dialogs during animation, or something like that